src/Illuminate/Session/SessionManager.php

```diff
@@ -103,7 +105,18 @@ protected function createApcDriver()
 	 */
 	protected function createMemcachedDriver()
 	{
-		return $this->createCacheBased('memcached');
+		$store = $this->app['config']['session.memcached_store'];
+
+		// Verify store uses the memcached driver
+		$repository = $this->app['cache']->store($store);
+		if (!($repository->getStore() instanceof MemcachedStore))

```

Comment:
I think Graham meant you don't need to wrap the instance check in brackets. Also, you need a space before and after `!`.